exchanges/kraken/kraken_test.go

```diff
@@ -1387,11 +1377,47 @@ func TestWsAddOrder(t *testing.T) {
 	}
 }
 
-func TestWsCancelOrder(t *testing.T) {
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, k, canManipulateRealOrders)
-	setupWs(t)
-	if err := k.wsCancelOrders([]string{"1337"}); err != nil {
-		t.Error(err)
+func TestWsCancelOrders(t *testing.T) {
+	t.Parallel()
+	if sharedtestvalues.AreAPICredentialsSet(k) && canManipulateRealOrders { // Live test
+		setupWs(t)
+		if err := k.wsCancelOrders([]string{"1337"}); err != nil {
+			t.Error(err)
+		}
+	} else {
+		k := wsMockInstance(t, func(msg []byte, w *websocket.Conn) error {
+			var req WsCancelOrderRequest
+			if err := json.Unmarshal(msg, &req); err != nil {
+				return err
+			}
+			if req.Event != krakenWsCancelOrder {
+				return w.WriteMessage(websocket.TextMessage, []byte(fmt.Sprintf(`{"event":"subscriptionStatus","reqid":%d,"status":"ok"}`, req.RequestID)))
+			}
+			resp := WsCancelOrderResponse{
+				Event:     krakenWsCancelOrderStatus,
+				Status:    "ok",
+				RequestID: req.RequestID,
+				Count:     int64(len(req.TransactionIDs)),
+			}
+			if len(req.TransactionIDs) == 0 || strings.Contains(req.TransactionIDs[0], "FISH") { // Reject anything that smells suspicious
+				resp.Status = "error"
+				resp.ErrorMessage = "[EOrder:Unknown order]"
+			}
+			json, err := json.Marshal(resp)
```

Comment:
json var -> marshalledData just so there isn't a package conflict.

Reply:
Yep, agreed.
I tend to avoid using "data" for blobs of things, because it's mostly meaningless.

Fixed 05ec2ed6